src/mincost.jl

```diff
@@ -1,18 +1,22 @@
 """
-    mincost_flow(graph, capacity, demand, cost, solver, [, source][, sink])
+    mincost_flow(graph, node_demand, edge_capacity, edge_cost, solver,<keyword argument>)
 
-Find a flow satisfying the `demand` and `capacity` constraints for each edge
-while minimizing the `sum(cost.*flow)`.
+Find a flow satisfying the `node_demand` at each node and `edge_capacity` constraints for each edge
+while minimizing the `sum(edge_cost.*flow)`.
 
-- If `source` and `sink` are specified, they are allowed a net flow production,
-consumption respectively. All other nodes must respect the flow conservation
-property.
+-`node_demand` is a vector of nodal values, which should be positive for sources and
+ negative at sink nodes, any node with zero demand is a transport node
 
-- The problem can be seen as a linear programming problem and uses a LP 
+- The problem can be seen as a linear programming problem and uses a LP
 solver under the hood. We use Clp in the examples and tests.
 
 Returns a flow matrix, flow[i,j] corresponds to the flow on the (i,j) arc.
-
+# Arguments
+- `edge_demand::AbstractMatrix`: demand a minimum flow for a given edge.
+- `edge_demand_exact=true`: changes the capacity of a non zero edge to the demanded value
+- `sources_to_maximize::AbstractVector` sources at which the nodal production will be >= node_demand
+- `sinks_to_maximize::AbstractVector` sinks at which the nodal consumption will be >= abs(node_demand),
+this is really going to minimize this the flo/
```

Comment:
"minimize this the flo/", I don't get the end of the sentence, what is meant there?

Reply:
Besides that typo at the end.   If you think about it sources have positive values in the demand matrix and sinks have negative demand values, so to get the maximum absolute flow through a sink you are minimizing the value.   So you could argue the argument should be called sinks_to_minimize.  I will try to improve the documentation